README.md

```diff
@@ -1,17 +1,44 @@
-# Google CloudEvents – Python
+# Google Events Library for Python
```

Comment:
I think we want to keep this title for now like other repos.

Reply:
Done.